fuse/core.go

```diff
@@ -177,8 +178,15 @@ func ServeFuseFS(filesys *plugin.Registry, mountpoint string) (chan<- bool, <-ch
 
 		log.Infof("FUSE: Serving filesystem")
 
+		serverConfig := &fs.Config{
+			WithContext: func(ctx context.Context, req fuse.Request) context.Context {
```

Comment:
That seems easy...

Reply:
Yup. Verified that it works by running a `stat` and `ls` and saw journal activity being recorded for them.